### DISCUSSION TITLE
Clearing browser cache after upgrade

### DISCUSSION THREAD

**Discussion by andrewcouling** [timschofield/webERP#271] [Development Discussion & Specification]
I have had users report issues following upgrade to 4.15.2 which were remedied by clearing their browser cache. The issues related to MiscFunctions.js and styles,  impacting message boxes, favourites,  date picker and icon images.

Would it be possible to add a message in the installer/upgrade script, prompting admins to clear their browser cache, and also a similar message which is shown to all users upon first login after an upgrade?

**Comment by timschofield** [comment]
Hi Andy, I have added ?version=1.0 to the end of the link statements for
styles.css and MiscFunctions.js to force a reload. This just needs to be
updated every time we do changes to the css or js.

Thanks
Tim

On Wed, 4 Dec 2024 at 13:41, Andrew Couling ***@***.***>
wrote:

> I have had users report issues following upgrade to 4.15.2 which were
> remedied by clearing their browser cache. The issues related to
> MiscFunctions.js and styles, impacting message boxes, favourites, date
> picker and icon images.
>
> Would it be possible to add a message in the installer/upgrade script,
> prompting admins to clear their browser cache, and also a similar message
> which is shown to all users upon first login after an upgrade?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/timschofield/webERP/discussions/271>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAL6LHELZJQBKPCVJVXSEL2D4BAHAVCNFSM6AAAAABTAGKNDSVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZXGYYTAMRZGQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
www.weberp.org <http://www.weberpafrica.com>
@TimSchofield2
Blog: https://kwamoja.home.blog/

**Reply by andrewcouling** [reply]
Thanks Tim.
Great solution. I was reading about this approach today while struggling to clear js and/or css from browser cache (even after hard reload).
